src/test/java/sg/dex/starfish/integration/squid/SquidAssetTests.java

```diff
@@ -146,7 +145,7 @@ public void testRegisterOnSurferAndChain() throws IOException, EthereumException
         String asset_metaData = new String(Files.readAllBytes(Paths.get("src/test/resources/assets/SJR8961K_metadata.json")));
 
         // create asset using metadata and given content
-        Asset memory_asset= ResourceAsset.create(asset_metaData,"assets/SJR8961K_content.json");
+        ResourceAsset memory_asset= ResourceAsset.create("assets/SJR8961K_content.json",JSON.toMap(asset_metaData));
```

Comment:
I think you should check with Aishwarya on whether you can use the SGCarmart content ( which I assume it is), since the content was not meant to be public (which it will become once this repo is made public).